ComprefaceTestApp/Services/ExampleSubjectService.cs

```diff
@@ -51,4 +58,59 @@ public async Task<ListAllExampleSubjectResponse> GetAllExampleSubjects(ListAllEx
 
         return response;
     }
+
+    public async Task<DeleteAllExamplesResponse?> ClearSubjectAsync(DeleteAllExamplesRequest request)
+    {
+        var requestUrl = $"{_httpClient.BaseAddress}recognition/faces";
+
+        var response = await requestUrl.
+            SetQueryParam("subject", request.Subject)
+            .DeleteAsync(HttpCompletionOption.ResponseContentRead);
```

Comment:
you can remove `HttpCompletionOption.ResponseContentRead` by default `completionOption` will be `HttpCompletionOption.ResponseContentRead`